packages/website/plugins/generated-rule-docs/insertions/insertFormattingNotice.ts

```diff
@@ -6,8 +6,8 @@ export function insertFormattingNotice(page: RuleDocsPage): void {
   if (page.rule.meta.type === 'layout') {
     page.spliceChildren(0, 0, {
       value: `
-<admonition type="warning">
-This rule will soon be moved to <a href="https://eslint.style">eslint-stylistic</a>.
+<admonition title="Deprecated" type="warning">
+This rule is deprecated. Formatting rules now live in <a href="https://eslint.style">eslint-stylistic</a>.
```

Comment:
can we do one better here and link directly to the replacement rule's docs?
I'd assume their URLs are gonna stay static so we can just hard link (cc @antfu so we can ensure redirects are in place should you restructure your URLs).

eg:
```ts
const replacement = page.rule.meta.replacedBy.find(e => e.startsWith('@stylistic/'));
const url = replacement ? `https://eslint.style/rules/default/${replacement.replace('@stylistic/', '')}` : 'https://eslint.style';
// ...
```